src/urg_stamped.cpp

```diff
@@ -447,12 +503,18 @@ void UrgStampedNode::cbRS(
     return;
   }
   scip2::logger::info() << "Sensor reset succeeded" << std::endl;
-  if (!device_initialized_)
+  if (failed_)
   {
-    device_initialized_ = true;
-    scip_->sendCommand("VV");
-    scip_->sendCommand("PP");
+    scip2::logger::info() << "Restarting urg_stamped" << std::endl;
+    device_->stop();
+    sleepRandom(0.5, 1.0);
+    ros::shutdown();
```

Comment:
Is it important to sleep after the call to `device_->stop()`? I noticed in https://github.com/seqsense/urg_stamped/blob/9a751b9635a2b794144764821261b6b48c566b15/src/urg_stamped.cpp#L482-L484 the sleep is before.

Reply:
On RS (soft reset) command, TCP connection will be kept. I think it's better to disconnect TCP connection by PC and then sleep in this case.

On RB (hard reset) command, sensor will be rebooted and the TCP connection will be disconnected by the sensor. In this case, `device_->stop()` doesn't actually do something effective, so the order is not actually meaningful.
It can be unified to `stop()` -> `sleep` order.